front/src/modules/timetableItem/components/ImportTimetableItem/helpers/parseXML.ts

```diff
@@ -1,15 +1,45 @@
+import { omit } from 'lodash';
+import { v4 as uuidV4 } from 'uuid';
+
 import type { CichDictValue, TimetableJsonPayload } from 'applications/operationalStudies/types';
-import type { PacedTrain, TrainSchedule } from 'common/api/osrdEditoastApi';
+import type { PacedTrain, TrainSchedule, PacedTrainException } from 'common/api/osrdEditoastApi';
 import { Duration } from 'utils/duration';
 
 import { buildSteps, cleanTimeFormat } from './buildStepsFromOcp';
 import findMostFrequentScheduleInPacedTrain from './findMostFrequentXmlSchedule';
+import { generatePacedTrainException } from '../../ManageTimetableItem/helpers/buildPacedTrainException';
+
+// These exceptions fields are not handled for now during import
+const fieldsToIgnore = [
+  'train_name',
+  'constraint_distribution',
+  'initial_speed',
+  'labels',
+  'options',
+  'rolling_stock_category',
+  'speed_limit_tag',
```

Comment:
Do we really need to specify all of these fields? If we always set them to `undefined` during the import, we shouldn't need to blacklist them I think, right?

I think we only need `train_name` here?

Dropping these would avoid missing exceptions for these if we start populating them in the import code.